src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -1141,6 +1170,9 @@ class IPRModelsDescription:
     linear_models: Dict[str, LinearIPRDescription] = attr.ib(
         default=attr.Factory(dict), validator=dict_of(LinearIPRDescription)
     )
+    vogel_models: Dict[str, VogelIPRDescription] = attr.ib(
+        default=attr.Factory(dict), validator=dict_of(VogelIPRDescription)
+    )
```

Comment:
I tried, but `inv cog` complains about it.

Reply:
Probably because we don't use type hints with `attrib_*` guys...
Given this class uses the a manual loader it is OK to directly use `attr.ib` I guess.